frontend/src/metabase/browse/components/BrowseDatabases.tsx

```diff
@@ -4,7 +4,6 @@ import { color } from "metabase/lib/colors";
 import * as Urls from "metabase/lib/urls";
```

Comment:
This small styling change is not related to the theme of this PR. Happy to extract it out.